crates/sui-types/src/sui_system_state/mod.rs

```diff
@@ -36,6 +38,8 @@ pub const ADVANCE_EPOCH_SAFE_MODE_FUNCTION_NAME: &IdentStr = ident_str!("advance
 
 #[cfg(msim)]
 const SUI_SYSTEM_STATE_SIM_TEST_V1: u64 = 18446744073709551605; // u64::MAX - 10
+#[cfg(msim)]
+const SUI_SYSTEM_STATE_SIM_TEST_V2: u64 = 18446744073709551606; // u64::MAX - 9
```

Comment:
nit: out of interest, why not define these directly as `u64::MAX - 10` and `u64::MAX - 9` instead of as the literal with the comment explaining?

Reply:
I am trying to make it obvious that the number is consistent as used in Move, and in Move I cannot write `u64::MAX - 10`.